src/app/features/vendor/vendor-listing/vendor-listing.component.ts

```diff
@@ -35,12 +35,12 @@ export class VendorListingComponent implements OnInit {
   vendors!: any[];
```

Comment:
remove any type and provide appropriate type.
Remove router if it is not used in the component